path-animate/demo.html

```diff
@@ -117,7 +117,45 @@
                         .drawCircle(0, 0, Math.max(this.width, this.height) * 0.6);
                     highlight.updateCache();
                 }
-            }
+            };
+
+            airplane.setPosition = function () {
+                var temp = Math.floor(Math.random() * 4); // determines which side airplane will enter from
+                var endpoint = [];
+
+                if (temp < 2) { // airplane coming from either top or below
+                    airplane.x = Math.floor(Math.random() * canvas.width);
+                    endpoint[0] = Math.floor(Math.random()*canvas.width);
+
+                    if (temp === 0) { // airplane coming from top
+                        airplane.y = -airplane.height;
+                        endpoint[1] = canvas.height + airplane.height;
+
+                    } else if (temp === 1) { // airplane coming from below
+                        airplane.y = canvas.height + airplane.height;
+                        endpoint[1] = -airplane.height;
+                    }
+                }
+
+                else if (temp >= 2) { // airplane coming from either right or left
```

Comment:
```suggestion
                else { // airplane coming from either right or left
```

Reply:
redundant code. if `temp` is not less than 2, then it must be greater than or equal to 2.